src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -271,24 +269,27 @@ def sync_deploy_root_with_stage(self, role: str) -> DiffResult:
             )
 
         # Perform a diff operation and display results to the user for informational purposes
-        log.info(
-            'Performing a diff between the Snowflake stage and your local deploy_root ("%s") directory.',
-            self.deploy_root,
+        cc.step(
+            "Performing a diff between the Snowflake stage and your local deploy_root ('%s') directory."
+            % self.deploy_root
         )
         diff: DiffResult = stage_diff(self.deploy_root, self.stage_fqn)
-        log.info("Listing results of diff:")
-        log.info("New files only on your local: %s", ",".join(diff.only_local))
-        log.info("New files only on the stage: %s", ",".join(diff.only_on_stage))
-        log.info(
-            "Existing files modified or status unknown: %s", ",".join(diff.different)
+        diff_results_info = (
+            "\nNew files only on your local: %s \nNew files only on the stage: %s \nExisting files modified or status unknown: %s \nExisting files identical to the stage: %s"
```

Comment:
Had to manually add in `\n` and strip leading and trailing whitespaces as `dedent` is interestingly not working as expected. Tried it out as part of the tests and `dedent` is not trimming the leading whitespaces.

```
def method():
      diff_results_info = dedent("""
      New files only on your local: 
      %s
      New files only on the stage:
      %s
      Existing files modified or status unknown: 
      %s
      Existing files identical to the stage: 
      %s
      """ % (
      "\n".join(["A", "B", "C"]),
      "\n".join(["A", "B", "C"]),
      "\n".join(["A", "B", "C"]),
      "\n".join(["A", "B", "C"])))
```

gave an output
```
message('\n            New files only on your local: \n            A\nB\nC\n            New files only on the stage:\n            A\nB\nC\n            Existing files modified or status unknown: \n            A\nB\nC\n            Existing files identical to the stage: \n            A\nB\nC\n')
```

Reply:
Not following. I'd probably just create an array and keep appending lines to it. Then, we can just join + display at the very end